packages/dev/core/src/Meshes/Builders/linesBuilder.ts

```diff
@@ -7,6 +7,7 @@ import type { FloatArray, Nullable } from "../../types";
 import { LinesMesh } from "../../Meshes/linesMesh";
 import type { Scene } from "../../scene";
 import { VertexBuffer } from "../../Buffers/buffer";
+import { Tools } from "../../Misc";

```

Comment:
Looks like there is a build error relating to this import

Reply:
Thanks for catching that, fixed it!